packages/server/lib/config.coffee

```diff
@@ -80,10 +106,12 @@ validationRules = {
   integrationFolder: v.isString
   numTestsKeptInMemory: v.isNumber
```

Comment:
we could make these predicates stronger by having method `v.isPositive` to make sure user provided value is > 0

Reply:
Sounds good to me. Seems out of scope for this PR though...